examples/stest.py

```diff
@@ -581,7 +644,7 @@ def make_tests():
           "--kappa", "0.01",
           "--num-samples", "2"],
          {"fine-mean-l1": 0.54961180496539375,
-          "max-p-error": 0.39696741537314961}]
+          "max-p-error": 0.39869063097389679}]
```

Comment:
@barker29 In some PR you raised the concern on this mismatch. Does this update match with test result on your machine (up to `1e-4`)? While working on this PR, I realize that the test tolerance on travis is `1e-2` while the default tolerance specified in CMakeLists.txt is `1e-4`, I guess that's why the test is passing on travis but not on our machines.